packages/@react-spectrum/s2/src/Icon.tsx

```diff
@@ -31,3 +33,45 @@ export interface IllustrationContextValue extends IconContextValue {
 
 export const IconContext = createContext<ContextValue<IconContextValue, SVGElement>>({});
 export const IllustrationContext = createContext<ContextValue<IllustrationContextValue, SVGElement>>({});
+
+export function createIcon(Component: ComponentType<SVGProps<SVGSVGElement>>, context: Context<ContextValue<IconContextValue, SVGElement>> = IconContext): FunctionComponent<IconProps> {
+  return (props: IconProps) => {
+    let ref = useRef<SVGElement>(null);
+    let ctx;
+    // TODO: remove this default once we release RAC and use DEFAULT_SLOT.
```

Comment:
Copied from old transformer. We should actually change this but not right before release.